src/commands/pets/PetFeedCommand.js

```diff
@@ -231,76 +230,70 @@
 						typeSuffix: pet.sex === PETS.FEMALE ? "se" : "x"
 					})
 				);
-			}
-			else {
+			} else {
 				successEmbed.setDescription(
 					format(tr.getTranslation(language).description["2"], {
 						petnick: await PetEntities.displayName(pet, language)
 					})
 				);
 			}
-		}
-		else {
+		} else {
 			guild[item.type]--;
 			successEmbed.setDescription(
 				format(tr.getTranslation(language).description["0"], {
 					petnick: await PetEntities.displayName(pet, language)
 				})
 			);
 		}
-	}
-	else {
+	} else {
 		pet.lovePoints += item.effect;
 		if (pet.lovePoints > PETS.MAX_LOVE_POINTS) {
 			pet.lovePoints = PETS.MAX_LOVE_POINTS;
 		}
 		guild[item.type]--;
 		switch (item.type) {
-		case "commonFood":
-			console.log(pet);
-			if (language === LANGUAGE.FRENCH) {
-				successEmbed.setDescription(
-					format(tr.getTranslation(language).description["1"], {
-						petnick: await PetEntities.displayName(pet, language),
-						typeSuffix: pet.sex === PETS.FEMALE ? "se" : "x"
-					})
-				);
-			}
-			else {
-				successEmbed.setDescription(
-					format(tr.getTranslation(language).description["1"], {
-						petnick: await PetEntities.displayName(pet, language)
-					})
-				);
-			}
-			break;
-		case "carnivorousFood":
-		case "herbivorousFood":
-			if (language === LANGUAGE.FRENCH) {
-				successEmbed.setDescription(
-					format(tr.getTranslation(language).description["2"], {
-						petnick: await PetEntities.displayName(pet, language),
-						typeSuffix: pet.sex === PETS.FEMALE ? "se" : "x"
-					})
-				);
-			}
-			else {
+			case "commonFood":
+				if (language === LANGUAGE.FRENCH) {
+					successEmbed.setDescription(
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 tabs but found 5.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7628754)
```suggestion
				successEmbed.setDescription(
```